exercises/practice/saddle-points/.meta/example.lua

```diff
@@ -0,0 +1,32 @@
+return function(matrix)
+  if #matrix == 0 or #matrix[1] == 0 then
+    return {}
+  end
+
+  local row_max = {}
+  local column_min = {}
+
+  for i, row in ipairs(matrix) do
+    row_max[i] = math.max(table.unpack(row))
+  end
+
+  for column = 1, #matrix[1] do
+    local column_values = {}
```

Comment:
# Naming convention consistency option A

For consistency with line 9, this could be

```
for j = ...
  local column = {}
```

Then line 23 would have

```
for i
  for j
```

# Naming convention consistency option B

Line 9 could have

```
for row, row_values
```

Line 15 could have
```
for _, row_values
```